app/views/partials/paymentAllocations/PaymentAllocationsCreditAmount.scala.html

```diff
@@ -40,14 +40,20 @@
     }
 }
 
+
 @paymentAllocations.paymentAllocationChargeModel.documentDetails.head.outstandingAmount.map { outstandingAmount =>
     @if(outstandingAmount != 0) {
          <tbody class="govuk-table__body">
                <tr id="money-on-account">
                    <td class="govuk-table__cell">
                        @getLink
                    </td>
-                   <td class="govuk-table__cell">@messages("paymentAllocation.na")</td>
+                   <td class="govuk-table__cell">
+                       @{paymentAllocations.paymentAllocationChargeModel
```

Comment:
the only fields is used in this view: => outstandingAmount and dueDate;
this would be a good idea to pass these as a parameters and rest: field validation, extraction on the controller level i.e

Reply:
Changes Made